src/main.cpp

```diff
@@ -540,7 +540,7 @@ void loop()
             strOutput += str_sprintf("LED FPS: %d ", g_Values.FPS);
 
             #if USE_STRIP
-                strOutput += str_sprintf("LED Bright: %f, LED Watts: %u, ", g_Values.Brite, g_Values.Watts);
+                strOutput += str_sprintf("LED Bright: %3.0lf%%, LED Watts: %d, ", g_Values.Brite, g_Values.Watts );

```

Comment:
I think the `%d` for `g_Values.Watts` should be a `%u`, because `g_Values.Watts` is a `uint32_t`.